common/changes/@itwin/manage-versions-react/vyki-revert-iui-update_2023-12-13-13-05.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/manage-versions-react",
+      "comment": "Revert iTwinUI v3",
+      "type": "none"
```

Comment:
```suggestion
      "type": "patch"
```

Reply:
yeah, had weird idea (written in description), but probab better release this as separate patch..

Reply:
> For manage-versions package made changes as none, thought https://github.com/iTwin/admin-components-react/pull/112 would be the one getting the patch.. But probably should make 2 separate releases anyway ?

Let's make 2 different releases, if simply to make it clear and facilitate the changes in #112 itself as there are already fixes attempts that may no longer be needed.